.github/workflows/yaml-lint.yaml

```diff
@@ -0,0 +1,13 @@
+name: reviewdog-yaml
+on: [pull_request]
+jobs:
+  yamllint:
+    name: runner / yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: yamllint
+        uses: reviewdog/action-yamllint@v1
+        with:
+          github_token: ${{ secrets.REVIEW_DOG }}
+          reporter: github-pr-review
```

Comment:
**[yamllint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[error] no new line character at the end of file (new-line-at-end-of-file)